Source/WebCore/platform/graphics/egl/GLContextEGL.cpp

```diff
@@ -161,9 +178,43 @@ bool GLContextEGL::getEGLConfig(EGLDisplay display, EGLConfig* config, EGLSurfac
 
 std::unique_ptr<GLContextEGL> GLContextEGL::createWindowContext(GLNativeWindowType window, PlatformDisplay& platformDisplay, EGLContext sharingContext)
 {
+    Function<bool(int)> checkCompatibleVisuals = nullptr;
+#if PLATFORM(X11)
+    XUniquePtr<XVisualInfo> visualInfo;
```

Comment:
Aha, I see the bug now. I think we can just transfer the ownership to the Function instead.